.vscode/launch.json

```diff
@@ -2,35 +2,6 @@
 {
   "version": "0.2.0",
   "configurations": [
-    {
```

Comment:
Hard to support two build setup, without bundling and with bundling. Anyway mostly we run bundled version with HMR, local service for HTML and etc. With some optimizations it takes not so long time (first build takes around 10-15s, then 2-3s)